applications/smart-distancing/libs/core.py

```diff
@@ -108,6 +113,8 @@ def calculate_distancing(self, objects_list):
         1. omitting large boxes by filtering boxes which are biger than the 1/4 of the size the image.
         2. omitting duplicated boxes by applying an auxilary non-maximum-suppression.
         3. apply a simple object tracker to make the detection more robust.
+        4. filter bounding boxes based on boxes prior distribution.
+        5. filter bounding boxes based on the amount of foreground pixels exist on the box.
 
```

Comment:
* Post processing step is getting complex and it's good to have a detailed documentation outside of code.
* I have an idea to develop a scoring algorithm for each bounding boxes based on these 5 steps criteria and detection score to omit extra bounding boxes in a unified way. each of steps add or reduce each bounding box acceptance score with a weighted score and bounding boxes with a score greater than a threshold will remain.